static/js/transform-facets.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * A Facets component transformFacets function which applies the configuration
+ * specified by the Facets 'fields' option
+ * 
+ * @param {DisplayableFacet[]} facets from answers-core
+ * @param {FilterOptionsConfig} config from answers-search-ui
+ * @returns {DisplayableFacet[]} from answers-core
+ */
+export default function transformFacets (facets, config) {
+  if(!config || !('fields' in config)) {
+    return facets;
+  }
+
+  return facets.map(facet => {
+    const isFacetFieldConfigSpecified = facet.fieldId in config.fields;
```

Comment:
What would you think of naming this `isConfigurationForFacet` or something like that? I think a name like that is a bit clearer: we want to know if the HHs have added special config for a particular facet.